kotlin/internal/jvm/compile.bzl

```diff
@@ -237,81 +267,174 @@ def kt_jvm_compile_action(ctx, rule_kind, output_jar, compile_jar):
     dirs = _compiler_directories(ctx)
     srcs = _partitioned_srcs(ctx.files.srcs)
     friend = _compiler_friends(ctx, friends = getattr(ctx.attr, "friends", []))
-    compile_deps = _compiler_deps(toolchains, friend, deps = ctx.attr.deps + ctx.attr.plugins)
+    compile_deps = _compiler_deps(toolchains, friend, deps = ctx.attr.deps)
```

Comment:
I removed plugins from compiler deps because `java_common.compile()` "discovers" plugins on the classpath and runs them, whether or not you specify `-proc:none`

Reply:
I think this is a good comment to live directly in the source code

Reply:
Agreed -- I did not know that, though it's the kind of shennigans I'd expect from java_common.

Reply:
This went in a previous PR actually - https://github.com/bazelbuild/rules_kotlin/pull/388 - plugins shouldn't be on the compile classpath anyway.

Reply:
Added a comment below when adding `-proc:none` to `javac_opts`